libminifi/include/utils/file/FileManager.h

```diff
@@ -17,18 +17,10 @@
 #ifndef LIBMINIFI_INCLUDE_UTILS_FILE_FILEMANAGER_H_
 #define LIBMINIFI_INCLUDE_UTILS_FILE_FILEMANAGER_H_
 
+#include <filesystem>
```

Comment:
I think these `std::filesystem`-related changes should be in a separate pull request.